Doc/library/tkinter.rst

```diff
@@ -95,6 +95,568 @@ Or, more often::
+      keyword arguments specify parameter of the event
+      (e.g. x, y, rootx, rooty).
+
+   .. method:: event_info(virtual=None)
+
+      Return a list of all virtual events or the information
+      about the SEQUENCE bound to the virtual event VIRTUAL.
+
+   Methods inheritted from :class:`WM` parent class:
+
+   .. method:: aspect(minNumer=None, minDenom=None, maxNumer=None, maxDenom=None)
+
+      Instruct the window manager to set the aspect ratio (width/height)
+      of this widget to be between MINNUMER/MINDENOM and MAXNUMER/MAXDENOM. Return a tuple
+      of the actual values if no argument is given.
+
+   .. method:: attributes(*args)
+
+      This subcommand returns or sets platform specific attributes
+
+      The first form returns a list of the platform specific flags and
+      their values. The second form returns the value for the specific
+      option. The third form sets one or more of the values. The values
+      are as follows:
+
+      On Windows, -disabled gets or sets whether the window is in a
+      disabled state. -toolwindow gets or sets the style of the window
+      to toolwindow (as defined in the MSDN). -topmost gets or sets
+      whether this is a topmost window (displays above all other
+      windows).
+
+      On Macintosh, XXXXX
+
+      On Unix, there are currently no special attribute values.
+
+   .. method:: client(name=None)
+
+      Store NAME in WM_CLIENT_MACHINE property of this widget. Return
+      current value.
+
+   .. method:: colormapwindows(*wlist)
+
+      Store list of window names (WLIST) into WM_COLORMAPWINDOWS property
+      of this widget. This list contains windows whose colormaps differ from their
+      parents. Return current list of widgets if WLIST is empty.
+
+   .. method:: command(value=None)
+
+      Store VALUE in WM_COMMAND property. It is the command
+      which shall be used to invoke the application. Return current
+      command if VALUE is None.
+
+   .. method:: deiconify()
+
+      Store VALUE in WM_COMMAND property. It is the command
+      which shall be used to invoke the application. Return current
+      command if VALUE is None.
+
+   .. method:: focusmodel(model=None)
+
+      Set focus model to MODEL. "active" means that this widget will claim
+      the focus itself, "passive" means that the window manager shall give
+      the focus. Return current focus model if MODEL is None.
+
+   .. method:: forget(window)
+
+      The window will be unmapped from the screen and will no longer
+      be managed by wm. toplevel windows will be treated like frame
+      windows once they are no longer managed by wm, however, the menu
+      option configuration will be remembered and the menus will return
+      once the widget is managed again.
+
+   .. method:: frame()
+
+      Return identifier for decorative frame of this widget if present.
+
+   .. method:: geometry(newGeometry=None)
+
+      Set geometry to NEWGEOMETRY of the form =widthxheight+x+y. Return
```

Comment:
Would be nice to mark up or otherwise differentiate the variables from the syntax. E.g. the TK manual uses italic _width_, _height_, _x_ and _y_.

Reply:
Minimal change:
```suggestion
      Set geometry to *newGeometry* of the form =widthxheight+x+y. Return
```